Private/Other/Get-ServersList.ps1

```diff
@@ -30,7 +31,11 @@ function Get-ServersList {
         }
         if ($Target.DomainControllers.Enabled) {
             if (-not $Quiet) { $Logger.AddInfoRecord("Preparing servers list - domain controllers autodetection") }
-            [Array] $Servers = (Get-WinADDomainControllers -SkipEmpty).HostName
+            if($Credential) {
```

Comment:
Isn't it always true? Since you set it to Empty by default?

Reply:
ok fixed.